### PR TITLE
Add More Ruff Checks

### PR DESCRIPTION
**Pull request by echavemann**
### Description

Title.

### Testing

CI
### Impact

devex

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,6 +26,17 @@ websockets = "^14.0"
...
+select = [
+    # isort
+    "I",
+    # pyflakes
+    "F",
+    # pycodestyle
+    "E",
+    "UP"
+]
+ignore = ["E501", "E402", "E741"]
```
Comment (rewong03) [inline pyproject.toml:39]: what is the rationale behind ignoring these

File: src/pysrc/adapters/kraken/spot/kraken_client.py [hunk 2]
```diff
@@ -1,19 +1,21 @@
+from typing import Any, Dict, Mapping, Optional
```
Comment (rewong03) [inline src/pysrc/adapters/kraken/spot/kraken_client.py:1]: 🤡 why'd we let aidan import Dict

File: pyproject.toml (hunk 1 above)
Comment (rewong03) [reply pyproject.toml:39]: i'm ok with e501 but why the others

Comment (echavemann) [reply pyproject.toml:39]: e402 ill flip, i use a custom improt scheme in the project I copied this from

Comment (echavemann) [reply pyproject.toml]: flipped

File: src/pysrc/adapters/kraken/spot/kraken_client.py (hunk 2 above)
Comment (echavemann) [reply src/pysrc/adapters/kraken/spot/kraken_client.py:1]: fml

Comment (echavemann) [reply src/pysrc/adapters/kraken/spot/kraken_client.py]: fixed

**Review by rewong03 — APPROVED** [review]

**Pull request merged**